Kwenta/kwenta
# Implement Funding Rate Time Period Toggle

## Description
Allow the user to select from a drop-down the time period they wish to view funding rate for.

## Related issue
#2148 

## Motivation and Context


## How Has This Been Tested?
1. Open the Futures page
2. Select different periods and see the change in the funding rate.

## Screenshots (if appropriate):
![image](https://github.com/Kwenta/kwenta/assets/4819006/eb05307b-6b7d-4bd3-abb3-a5b6e963ada1)
![image](https://github.com/Kwenta/kwenta/assets/4819006/9418a344-5ca5-48a3-a0c3-1e50202d338f)
